### PR TITLE
Add flag to server cmd to disable TLS

### LINKED ISSUE #985: Server mode: Add support for serving API over HTTP
Currently, the server can only be accessed via HTTPS. This makes it cumbersome to work with during dev and test.

Add support for accessing the API via HTTP. This should be disabled by default.

### PR DESCRIPTION
**Pull request by alexbrand**
Fixes #985

### CODE REVIEW

**Comment by dkoshkin** [comment]
Do you think its ok to keep the default port `443` [here](https://github.com/apprenda/kismatic/blob/c6366285f6d7ef324a5c6b85ff6d211fdf71d286/pkg/cli/server.go#L60)?

**Comment by alexbrand** [comment]
Thanks @dkoshkin. Changed the defaults.

File: pkg/cli/server.go [hunk 1]
```diff
@@ -94,10 +96,15 @@ func doServer(stdout io.Writer, options serverOptions) error {
 	// create handlers
 	clusterAPI := handler.Clusters{Store: clusterStore, AssetsDir: assetsDir, Logger: logger}
 
+	port := "8443"
```
Comment (dkoshkin) [inline pkg/cli/server.go]: Don't we also need to check if `options.port` is not empty and use that?

Comment (alexbrand) [reply pkg/cli/server.go]: Whoops! Yes, I basically rendered that flag useless.

Comment (dkoshkin) [reply pkg/cli/server.go]: :) I got you

**Comment by dkoshkin** [comment]
LGTM!

**Comment by emmetthitz** [comment]
LGTM

**Pull request merged**